.github/workflows/documentation.yml

```diff
@@ -0,0 +1,64 @@
+name: PMM Docs 3.x
+
+on:
+  push:
+    branches:
+      - main
+      - v3
+    paths:
+      - "documentation/**"
+  pull_request:
+    paths:
+      - "documentation/**"
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0 # read more about why this is necessary at https://github.com/jimporter/mike/issues/60
+          sparse-checkout: |
+            .github
+            documentation
+            Makefile
+            Makefile.include
+
+      - name: Setup Python
+        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
+        with:
+          python-version: '3.x'
+          cache: 'pip'
+
+      - name: Setup MkDocs
+        working-directory: documentation
+        run: |
+          python -m pip install --upgrade pip
+          pip install wheel
+          test -f requirements.txt && pip install -r requirements.txt
+
+      - name: Configure git
+        env:
+          ROBOT_TOKEN: ${{ secrets.ROBOT_TOKEN }}
+        run: |
+          git config --global url."https://percona-platform-robot:${ROBOT_TOKEN}@github.com".insteadOf "https://github.com"
+          git config user.name "GitHub Action"
+          git config user.email "github-action@users.noreply.github.com"
+          git config user.password "${ROBOT_TOKEN}"
+          echo "GIT_USER=percona-platform-robot:${ROBOT_TOKEN}" >> $GITHUB_ENV
+
+      - name: Check for unused images
+        run: make doc-check-images
+
+      # Builds Material-themed static web site in 'publish' branch
```

Comment:
it should run only for main and v3 branches, but not for PRs

Reply:
Absolutely! It was necessary to enforce the workflow for this `migration`, but otherwise we don't need it. 

Removed. Also, the `main` branch has its own workflow, so I removed it as well.